sklearn/multioutput.py

```diff
@@ -513,15 +513,15 @@ class MultiOutputClassifier(ClassifierMixin, _MultiOutputEstimator):
     def __init__(self, estimator, *, n_jobs=None):
         super().__init__(estimator, n_jobs=n_jobs)
 
-    def fit(self, X, Y, sample_weight=None, **fit_params):
+    def fit(self, X, y, sample_weight=None, **fit_params):
```

Comment:
We need to deprecated `Y` away instead of simply renaming it, to avoid breaking code which was written as `est.fit(X=X, Y=y)`

We then need to handle the deprecation also inside the function. Have a look at our contributing guides regarding deprecations.

```suggestion
    @_deprecate_positional_args(version="1.9")
    def fit(self, X, y, *, sample_weight=None, Y=None, **fit_params):
```